kit/inputs/src/UISeedPhraseTextView/UISeedPhrasePopover.tsx

```diff
@@ -21,8 +13,11 @@ import { Portal, UILayoutConstant } from '@tonlabs/uikit.layout';
 
 const MAX_CELLS = 3;
 
+const statusBarHeight =
+    Platform.OS === 'android' && StatusBar.currentHeight ? StatusBar.currentHeight : 0;
```

Comment:
I don't think that you should check for platform, as this constant works only for Android anyway

Reply:
This is a protection against possible future changes if this constant returns something for ios as well. I think it won't be superfluous.

Reply:
It actually never worked for iOS :) Let's not make it overcomplicated, if it will be changed it will break a lot of places not only this one :)

Reply:
Done